content/cloud-computing/concepts/cloud-services/cloud-services.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: 'Cloud Services'
+Description: 'There are many different cloud-computing services, but the big three are Microsoft Azure, Amazon Web Services (AWS) and Google Cloud Platform (GCP).'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Networking'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are many different **cloud-computing services**, but the big three are [Amazon Web Services (AWS)](https://aws.amazon.com/), [Microsoft Azure](https://azure.microsoft.com/), and [Google Cloud Platform (GCP)](https://cloud.google.com/). All three offer services to cover the most significant cloud-computing needs:
+
+- [Infrastructure as a service (IaaS)](https://www.codecademy.com/resources/docs/cloud-computing/iaas)
+- [Platform as a service (PaaS)](https://www.codecademy.com/resources/docs/cloud-computing/paas)
+- [Function as a service (FaaS)](https://www.codecademy.com/resources/docs/cloud-computing/faas)
+- [Software as a service (SaaS)](https://www.codecademy.com/resources/docs/cloud-computing/saas)
+
+## AWS
+
+AWS is the largest cloud computing platform, and the most mature, introducing its first public cloud services in 2006. It serves millions of users, from individuals to large enterprises, to governments.
+
+- AWS offers more than 200 services to its customers.
+- As of 2021 AWS has 25 geographic regions and 218+ edge locations.
+
+The advantages of AWS include:
+
+- The widest array of services; from networking to quantum computing.
+- The most mature, reliable and secure platform.
+- The most compute capability.
+
+Disadvantages include:
+
+- If Dev/Enterprise support is needed, users must pay for it separately.
+- The number of services and options can be overwhelming.
+- A complex and hard-to-understand pricing structure.
+- There are limited options to run a [hybrid cloud](https://www.codecademy.com/resources/docs/cloud-computing/types-of-cloud) solution.
+
+## Azure
+
+Azure is the second largest cloud platform, introduced my Microsoft in 2008. It is also currently among the fastest growing cloud platforms.
```

Comment:
"Azure is the second largest cloud platform, introduced ~my~ by Microsoft in 2008."